### ISSUE TITLE
Questions regarding M3's warmup and finetuning steps

### ISSUE THREAD

**Issue by umarbutler** [FlagOpen/FlagEmbedding#1499]
Hi @staoxiao @hanhainebula,
You state in your paper, 'In the initial phase, we employed approximately 6000 steps to perform warm-up on dense embedding, sparse embedding and multi-vectors. Subsequently, we conducted unified training with self-knowledge distillation' (B.1).

@hanhainebula, you also stated in #942, 'We performed pretraining for 1 epoch. For the finetuning process, we performed 2 epochs for warm-up (1 epoch for dense embedding, 1 epoch for sparse embedding and multi-vectors), and 1 epoch for unified-finetuning.'.

 I'd to understand:
1. When you say you trained for an epoch, how many steps was that? And were the epochs on the same finetuning data or different data?
2. When you say that you trained '1 epoch for sparse embedding and multi-vectors', were the dense embeddings also getting trained at the same time? Did you freeze the base encoder's weights during that warmup?
